ql/src/java/org/apache/hadoop/hive/ql/parse/repl/CopyUtils.java

```diff
@@ -222,9 +222,13 @@ private boolean isSourceFileMismatch(FileSystem sourceFs, ReplChangeManager.File
           verifySourceChecksumString
                   = ReplChangeManager.checksumFor(srcFile.getSourcePath(), sourceFs);
         } catch (IOException e) {
-          // Retry with CM path
-          LOG.debug("Unable to calculate checksum for source file: " + srcFile.getSourcePath());
-          return true;
+          LOG.info("Unable to calculate checksum for source file: " + srcFile.getSourcePath(), e);
```

Comment:
No need of info here. Can this be error log message only if we throw exception below?